### PR TITLE
chore(ui): Visual update summary area on call views

### PR DESCRIPTION
**Pull request by m-rgba**
## Description

| Before    | After |
| -------- | ------- |
| Screenshot 2024-11-21 at 8 57 39 AM  | Screenshot 2024-11-21 at 1 19 02 PM |

- Design update for the Summary table in calls.
- Also adds some styling to the `SimpleKeyValueTable` component.

## Testing

Tested visual changes on call / eval views.
`SimpleKeyValueTable` which was updated is also used in:
- https://github.com/wandb/weave/blob/3f43e8121e623efe3d9093f8b60b954254b50dff/weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OpVersionPage.tsx#L16
- https://github.com/wandb/weave/blob/3f43e8121e623efe3d9093f8b60b954254b50dff/weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionPage.tsx#L25

I didn't see it on those pages but the styling changes shouldn't cause any issues.

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx [hunk 1]
```diff
@@ -19,11 +20,19 @@ export const CallSummary: React.FC<{
   call: CallSchema;
 }> = ({call}) => {
   const span = call.rawSpan;
-  const attributes = _.fromPairs(
-    Object.entries(span.attributes ?? {}).filter(
-      ([k, a]) => !k.startsWith('_') && a != null
-    )
+
+  // Process attributes, only filtering out null values and keys starting with '_'
```
Comment (jwlee64) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx:24]: was this just refactored for clarity?

**Review by jwlee64 — APPROVED** [review]
nice lgtm

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx (hunk 1 above)
Comment (m-rgba) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx:24]: Mostly cause I ran over it a few times - did a couple of implementations that didn't work, but when I backed them out I was able to read this a bit better.

**Comment by adamwdraper** [comment]
@m-rgba i am seeing the model row in the details show, even when i don't have a model: https://beta.wandb.ai/wandb-designers/winston/weave/traces?cols=%7B%22attributes.weave.client_version%22%3Afalse%2C%22attributes.weave.os_name%22%3Afalse%2C%22attributes.weave.os_release%22%3Afalse%2C%22attributes.weave.os_version%22%3Afalse%2C%22attributes.weave.source%22%3Afalse%2C%22attributes.weave.sys_version%22%3Afalse%7D&peekPath=%2Fwandb-designers%2Fwinston%2Fcalls%2F01934bd3-ddea-7102-bd81-7da787b13ff7%3Fpath%3Dwinston-solve*0%26tracetree%3D1
Screenshot 2024-11-22 at 9 13 21 AM

**Pull request merged**